rdr_service/main.py

```diff
@@ -357,6 +358,8 @@ def _log_request_exception(sender, exception, **extra):  # pylint: disable=unuse
 # Data generator API used to load fake data into the database.
 api.add_resource(DataGenApi, API_PREFIX + "DataGen", endpoint="datagen", methods=["POST", "PUT"])
 
+# Message broker API
+api.add_resource(MessageBrokerApi, API_PREFIX + "Message/Broker", endpoint="message.broker", methods=["POST"])
```

Comment:
Is this the path naming convention that was decided on? Are there going to be more endpoints to the message API route?

Reply:
Based on current design, this is the only endpoint for message broker, we will use different `eventType` in the payload for different events.

Reply:
Should we make it `MessageBroker` then?

Reply:
Done